src/ontology/imports/obi_import.owl

```diff
@@ -58,10 +58,6 @@
         <obo:IAO_0000111 xml:lang="en">definition</obo:IAO_0000111>
         <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000122"/>
     </owl:AnnotationProperty>
-    <owl:AnnotationProperty rdf:about="http://www.w3.org/2000/01/rdf-schema#label">
```

Comment:
no need to delete this here.  It would simply get regenerated in next ontofox fetch.

Reply:
Yeah, I did that when I was doing some testing before our previous discussion. I figure I would just leave it and let it reappear on the next make.